### PR TITLE
glibmm 2.44.0

### PR DESCRIPTION
**Pull request by tschoonj**
version bump
passed brew audit --strict after adding test block

### CODE REVIEW

**Comment by tschoonj** [comment]
@mikemcquaid Is there a way that I could make the testbot allow me to use pkg-config in a test block?

**Comment by dunn** [comment]
I think `#{Formula["pkg-config"].opt_bin}/pkg-config --cflags sigc++-2.0` might work.

**Comment by jacknagel** [comment]
Mike has objections to tests requiring build dependencies, so if we can write this without pkg-config that would be good.

File: Library/Formula/glibmm.rb [hunk 1]
```diff
@@ -18,6 +16,21 @@ class Glibmm < Formula
 
   def install
     system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make install"
+    system "make"
+    system "make", "install"
```
Comment (jacknagel) [inline Library/Formula/glibmm.rb]: Is this change strictly required? Should it be reported upstream?

**Comment by tschoonj** [comment]
Hi Jack,

I am working my way up through the gtkmm3 dependency tree, updating the formulas wherever necessary. I found already 6 packages that need updating to their latest versions. I am also adding tests wherever necessary. However, I think it would be a good idea to use pkg-config for these tests because

1) If not, I would end up copy pasting the output of the pkg-config command into the test anyway. The output is very long for packages like gtk3 and would make things look messy
2) Tests would break when a package it relies on changes its header paths or linker flags. Not often, but it has been known to happen
3) pkg-config is a build requirement of these packages anyway. It should be present whenever one decides to run brew test

I am on holidays for the next four days now but I will continue working on this when I am back

Op 18-apr.-2015 om 04:12 heeft Jack Nagel <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


Mike has objections to tests requiring build dependencies, so if we can write this without pkg-config that would be good.

**Comment by mikemcquaid** [comment]
> 1) If not, I would end up copy pasting the output of the pkg-config command into the test anyway. The output is very long for packages like gtk3 and would make things look messy
> 2) Tests would break when a package it relies on changes its header paths or linker flags. Not often, but it has been known to happen

I'd rather we did this and fix the tests.

> 3) pkg-config is a build requirement of these packages anyway. It should be present whenever one decides to run brew test

That's not the case as it's not a runtime requirement so not installed when using a `bottle`.

File: Library/Formula/glibmm.rb (hunk 1 above)
Comment (jacknagel) [inline Library/Formula/glibmm.rb]: Otherwise, just change these two lines to just `system "make", "install"`

Comment (tschoonj) [inline Library/Formula/glibmm.rb]: I am on holiday right now. Will make the necessary changes on Wednesday



Op 19-apr.-2015 om 23:21 heeft Jack Nagel <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


In Library/Formula/glibmm.rb<https://github.com/Homebrew/homebrew/pull/38741#discussion_r28657041>:

> @@ -18,6 +16,21 @@ class Glibmm < Formula
>
>    def install
>      system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
> -    system "make install"
> +    system "make"
> +    system "make", "install"


Otherwise, just change these two lines to just system "make", "install"

**Comment by tschoonj** [comment]
Everything looks good now: I removed the `system "make"` line as well as all references to pkg-config in the test block. Next I will look into cairomm

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!
